src/main/scala/cromwell/engine/store/StoreActor.scala

```diff
@@ -24,9 +32,9 @@ object StoreActor {
 /**
  * Actor to hold symbol and execution status data for a single workflow.  This actor
  * guards mutable state over the symbol and execution stores. */
-class StoreActor(namespace: WdlNamespace, hostInputs: HostInputs) extends Actor with CromwellActor {
-  private val symbolStore = new SymbolStore(namespace, hostInputs)
-  private val executionStore = new ExecutionStore(namespace)
+ class StoreActor(namespace: WdlNamespace, hostInputs: HostInputs, initialStore: InitialStore) extends Actor with CromwellActor {
+  private val symbolStore = new SymbolStore(namespace, hostInputs, initialStore.symbolStore)
```

Comment:
I'm going to change both of these such that the StoreActor is passed the built xyzStore and there'll be diff Props to handle the different configurations